sqlite/src/comdb2lua.c

```diff
@@ -21,6 +21,7 @@ struct dbtable;
 struct dbtable *getqueuebyname(const char *);
 int bdb_get_sp_get_default_version(const char *, int *);
 
+#define LUA_TAG_LEN 3
```

Comment:
Does this name make sense?

Reply:
Should be fine. Could call `Q4SP` instead, and check if `strlen` of resulting queue-name is less than max tablename-length. That way, we encode the existence of `__q` in one single place.

Reply:
Is [this commit](https://github.com/bloomberg/comdb2/pull/5009/commits/df7567645f039ef02e6a30f13ca7f85268f23424) an improvement? I extracted `__q` into a macro to compute the length limit for the error message instead of hardcoding it.